charts/rancher-istio/1.5.920/Chart.yaml

```diff
@@ -14,4 +14,4 @@ name: rancher-istio
 sources:
```

Comment:
Odd that the original folder here was 1.5.902 instead of 1.5.901, guessing that was folder name of copy, and that would never go public.

Reply:
Correct - in the first commit I did was a copy of 1.5.901 and then rename it to 1.5.902. In the second commit I made the changes I needed to and that included the rename to 1.5.920 so that is why its showing up that way.